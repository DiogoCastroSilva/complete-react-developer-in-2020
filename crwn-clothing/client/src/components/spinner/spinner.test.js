import React from 'react';
import { shallow } from 'enzyme';

import Spinner from './spinner.component';


describe('Testing Spinner component', () => {

    it('expects to render Spinner component', () => {
        expect(shallow(<Spinner />).length).toEqual(1);

        expect(shallow(<Spinner />)).toMatchSnapshot();
    });

});
