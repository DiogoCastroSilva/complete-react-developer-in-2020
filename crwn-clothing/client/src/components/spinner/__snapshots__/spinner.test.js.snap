// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Spinner component expects to render Spinner component 1`] = `
<styled.div>
  <styled.div />
</styled.div>
`;
